contrib/themes-megapack/README.org

```diff
@@ -9,6 +9,8 @@
 This simple contribution layer is an example. It installs around 100 themes
 for Emacs. You can try them easily by invoking helm-themes with: ~<SPC> T h~.
 
+You can see samples of all included themes in this [[http://themegallery.robdor.com][theme gallery]].
+
```

Comment:
Add your github profile or twitter, something like:
You can see samples of all included themes in this theme gallery from robmerrell